app/components/support_interface/application_choice_component.rb

```diff
@@ -48,7 +48,15 @@ def rows
         rows << { key: 'API', value: 'This application hasn’t been sent to the provider yet, so it isn’t available over the API' }
       end
 
-      rows
+      if application_choice.interviews.present?
+        interview_blocks = application_choice.interviews.order('created_at').map do |interview|
+          render(SupportInterface::InterviewDetailsComponent.new(interview: interview))
+        end
+
+        rows << { key: 'Interviews', value: interview_blocks }
+      end
+
+      rows.flatten
```

Comment:
```suggestion
      rows
```

There was an issue on line 22 resolved [here](https://github.com/DFE-Digital/apply-for-teacher-training/commit/98c16ed80bb9b398c08360a37f6ff7f8f3ecf021) so we shouldn't need to flatten the rows.

Reply:
Cool, removed